src/runtime_src/core/common/api/hw_queue.cpp

```diff
@@ -267,9 +267,9 @@ class kds_device
   // execution. The execution monitor is by-passed and will be unaware
   // of argument command having been scheduled for execution.
   void
-  exec_buf(xrt_core::command* cmd)
-  {
-    device->exec_buf(cmd->get_exec_bo());
+  exec_buf(xrt_core::command* cmd, const xrt::hw_context& hwctx)
+  {    
+    device->exec_buf_ctx(cmd->get_exec_bo(), hwctx);
```

Comment:
Sorry, I didn't catch this the first time around.   This is C++, I would suggest we simply overload exec_buf.
```suggestion
    device->exec_buf(cmd->get_exec_bo(), hwctx);
```

Reply:
fixed.